website/docs/docs/cloud/manage-access/set-up-bigquery-wif.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Set up BigQuery WIF"
+description: "Learn how configure dbt Cloud and BigQuery to use workload identity federation"
+id: "set-up-bigquery-wif"
+pagination_next: null
+---
+
+# Set up BigQuery workload identity federation <Lifecycle status='beta,managed'/> 
+
+Workload identity federation (WIF) allows application workloads, running externally to dbt Cloud, to act as a service account without the need to manage service accounts or other keys. The following instructions will enable you to authenticate your BigQuery connection in dbt Cloud using WIF. 
+
+## Set up dbt Cloud
+
+To configure a BigQuery connection to use WIF authentication in dbt Cloud, you must set up a custom OAuth integration configured with details from the Entra application used as your workpool provider in GCP.
```

Comment:
I'm a little nervous about writing this just about Entra because eventually we will bring support for other IdPs and it'll take more effort to update this doc